### PR TITLE
Fix D2IQ-71726 and D2IQ-69008

### PR DESCRIPTION
**Pull request by ianhellstrom**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [X] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/kubeflow/1.0.1-0.5.0/install/konvoy/index.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ enterprise: false
...
+* [Download `kubeflow-1.0.1_0.5.0.tgz` tarball](../../download/).
 * Install Kubeflow:
   ```bash
-  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1-0.5.0.tgz
+  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1_0.5.0.tgz
   ```
 * If you would like to inject additional annotations to KUDO for Kubeflow's default gateway `kubeflow-ingressgateway`, you can pass in the service annotations as parameters:
   ```bash
-  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1-0.5.0.tgz -p kubeflowIngressGatewayServiceAnnotations='{"foo": "abc","bar": "xyz"}'
+  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1_0.5.0.tgz -p kubeflowIngressGatewayServiceAnnotations='{"foo": "abc","bar": "xyz"}'
```
Comment (porridge) [inline pages/ksphere/kubeflow/1.0.1-0.5.0/install/konvoy/index.md:22]: Please revert these changes. They do not apply to `0.5.0`.

**Review by porridge — APPROVED** [review]
LGTM apart from one piece which needs to be reverted for now.

File: pages/ksphere/kubeflow/1.0.1-0.5.0/install/konvoy/index.md (hunk 1 above)
Comment (ianhellstrom) [reply pages/ksphere/kubeflow/1.0.1-0.5.0/install/konvoy/index.md]: Done

**Comment by ianhellstrom** [comment]
@cmastr The issue with "Pipelines" not showing up in the tutorials appears fixed when I build it locally. It should be ready for your review.

**Review by cmastr — APPROVED** [review]
